emails/templates/ng_alert_notification.txt

```diff
@@ -0,0 +1,47 @@
+[[Subject .Subject "[[.Title]]"]]
+
+[[.Title]]
+----------------
+
+[[ .Alerts | len ]] alert[[ if gt (len .Alerts) 1 ]]s[[ end ]] for
+[[ range .GroupLabels.SortedPairs ]]
+[[ .Name ]]=[[ .Value ]]
+[[ end ]]
+
+[[ if gt (len .Alerts.Firing) 0 ]]
+([[ .Alerts.Firing | len ]]) Firing
+[[ end ]]
+[[ range .Alerts.Firing ]]
+Labels:
+[[ range .Labels.SortedPairs ]]
+[[ .Name ]] = [[ .Value ]]
+[[ end ]]
+[[ if gt (len .Annotations) 0 ]]
+Annotations:
+[[ end ]]
+[[ range .Annotations.SortedPairs ]]
+[[ .Name ]] = [[ .Value ]]
+[[ end ]]
+[[ end ]]
+
+[[ if gt (len .Alerts.Resolved) 0 ]]
```

Comment:
If any of the `if` condition becomes false, it will still end up adding a newline to the final render. An example of how we can avoid those newlines is [here](https://github.com/grafana/grafana/blob/f0cac149dacfff9be191161979e083daa2f91821/pkg/services/ngalert/notifier/channels/default_template.go#L15-L24), where you don't start `if` block on a new line.


The alert that I got:
```
[FIRING:1]  (AlwaysFiring value)
----------------

1 alert for



(1) Firing


Labels:

alertname =3D AlwaysFiring

key =3D value








View your Alert rule:
http://localhost:3000/alerting/list

Go to the Alerts page:
http://localhost:3000/alerting/list?alertState=3Dfiring&amp;view=3Dstate

Sent by Grafana v8.1.0-pre (c) 2021 Grafana Labs
```

Reply:
Updated